tests/test_auto_captcha.py

```
@@ -105,7 +105,13 @@ async def close(self):
         self.closed = True
 
     # Only post is used by _handle_captcha_challenge
-    def post(self, url: str, headers: Optional[Dict] = None, data: Optional[Dict] = None, timeout: int = 30):
+    def post(
+        self,
+        url: str,
```

Ruff (0.14.13)

Unused method argument: `url`

(ARG002)